showItemPageBE/src/test/java/com/talentpath/shamazin/showItemPage/services/ItemServiceTest.java

```diff
@@ -24,33 +21,32 @@
 
     @Autowired
     ItemService itemServe;
-    @Autowired
-    ItemRepository itemDao;
-    @Autowired
-    ItemFamilyRepository itemFamilyDao;
-    @Autowired
-    ItemFamilyService itemFamilyService;
+
     @BeforeEach
-    void setUp() {
-        itemFamilyService.truncateItemFamily();
+    void deleteAllItems() {
+        itemServe.deleteAllRows();
```

Comment:
This won't reset the sequences, I would keep it the way it was.

Reply:
good idea @neveland98 . @jlavendtt ignore my strikethrough comment above, i think this suggestion's the way to go